docker/Dockerfile.dev

```diff
@@ -55,7 +55,7 @@ COPY . .
 RUN sudo rm -rf build \
     && mkdir build \
     && cd build \
-    && cmake -DCMAKE_BUILD_TYPE=Debug .. \
+    && cmake .. \
```

Comment:
why change this?

The dockerfile is meant for development and I often use GDB from inside it, so having a build with symbols is often good.

Though, I usually attach a folder from host and rebuild the DB for debugging so not a big deal but would still like to know why the change.

Reply:
The build did not succeed for me without it